Example/StatusViewController.swift

```diff
@@ -61,7 +61,7 @@ class StatusViewController: UIViewController {
     }
 
     @IBAction func didClickPasswordlessPhoneLogin(_: Any) {
-        UIApplication.identityUI.presentIdentityProcess(from: self, loginMethod: .phone)
+        UIApplication.identityUI.presentIdentityProcess(from: self, loginMethod: .phone, scopes: ClientConfiguration.current.scopes)
```

Comment:
Should the scopes be injected into the controller instead of using a singleton?

Reply:
Ah, that's just an extension for the example app.